client/web/src/components/diff/DiffStat.tsx

```diff
@@ -47,24 +47,33 @@ export const DiffStat: React.FunctionComponent<DiffStatProps> = React.memo(funct
         labels.push(`${numberWithCommas(deleted)} ${pluralize('deletion', deleted)}`)
     }
     return (
-        <div className={classNames(styles.diffStat, className)} data-tooltip={labels.join(', ')}>
-            {expandedCounts ? (
-                <>
-                    {/*
+        <>
+            <div className="sr-only">{`Diff stat: added ${added} ${pluralize(
+                'line',
+                added
+            )}, changed ${changed} ${pluralize('line', changed)}, deleted ${deleted} ${pluralize(
+                'line',
+                deleted
+            )}`}</div>
```

Comment:
This class comes from [Bootstrap](https://getbootstrap.com/docs/4.0/utilities/screenreaders/), correct? My understanding is we're slowly phasing Bootstrap out, so I might suggest using `<VisuallyHidden />` here instead to achieve the same result. From inspecting our repo, usage of `.sr-only` are few and far between ([5 results across 3 files](https://sourcegraph.com/search?q=context:%40sourcegraph+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+sr-only+lang:typescript+&patternType=literal)), compared to `<VisuallyHidden />` ([41 results across a dozen files](https://sourcegraph.com/search?q=context:%40sourcegraph+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+VisuallyHidden+lang:typescript+&patternType=literal)).

I think I will open a docs PR to add a note about this to our auditing instructions. 🙂

```suggestion
            <VisuallyHidden>{`Diff stat: added ${added} ${pluralize(
                'line',
                added
            )}, changed ${changed} ${pluralize('line', changed)}, deleted ${deleted} ${pluralize(
                'line',
                deleted
            )}`}</VisuallyHidden>
```

Reply:
Yep! I was obviously unaware of `VisuallyHidden`, so I'll transform all the `sr-only` uses into that. Thanks for pointing that out!